README.md

```diff
@@ -64,19 +71,42 @@ hash. A sample configuration file might look like:
 
 ```javascript
 {
+  "esclient": {
+    "hosts": [
+      {
+        "env": "development",
+        "protocol": "http",
+        "host": "localhost",
+        "port": 9200
+      }
+    ]
+  },
+  "logger": {
+    "level": "debug"
+  },
   "imports": {
+    "whosonfirst": {
+      "datapath": "/mnt/data/whosonfirst/",
+      "importPostalcodes": false,
+      "importVenues": false
+    },
     "openaddresses": {
-        "deduplicate": false,
-        "datapath": "/tmp/oa-data",
-        "files": ["us/ny/city_of_new_york.csv"]
-      }
+      "adminLookup": true,
+      "datapath": "/mnt/data/openaddresses/",
+      "files": [ "us/ny/city_of_new_york.csv" ]
     }
+  }
 }
 ```
 
 The following properties are recognized:
-  * `deduplicate` : Boolean flag to enable deduplication (see above).
-  * `datapath`: The absolute path of the directory containing OpenAddresses files. Must be specified if no directory is
-    given as a command-line argument.
-  * `files`: An array of the names of the files to import. If specified, *only* these files will be imported, rather
-    than *all* `.csv` files in the given directory. **If the array is empty, all files will be imported.**
+
+This importer is configured using the [`pelias-config`](https://github.com/pelias/config) module.
+The following configuration options are supported by this importer.
+
+| key | required | default | description |
+| --- | --- | --- | --- |
+| `deduplicate` | no | `false` | Boolean flag to enable deduplication (see above). |
+| `adminLookup` | no | `true` | Boolean flag to enable admin-lookup. |
+| `datapath` | yes | | The absolute path of the directory containing OpenAddresses files. Must be specified if no directory is given as a command-line argument. |
+| `files` | no | | An array of the names of the files to download/import. If specified, *only* these files will be downloaded and imported, rather than *all* `.csv` files in the given directory. **If the array is empty, all files will be downloaded and imported.** |
```

Comment:
Someone new to OpenAddresses might not know how the files or directories are named. Add a link to http://results.openaddresses.io/?runs=all#runs here or under "Data Download"